confluence/models/content.py

```diff
@@ -83,6 +83,9 @@ def __init__(self, json):  # type: (Dict[str, Any]) -> None
         if 'version' in json:
             self.version = Version(json['version'])
 
+        if '_links' in json:
```

Comment:
Can the json ever not have _links? If it's mandatory (I think it is) then we shouldn't have this check. That way we don't need to be defensive later on about whether content.links exists wherever it's used.